Scribe_API.postman_collection.json

```diff
@@ -2,7 +2,8 @@
 	"info": {
 		"_postman_id": "a347be16-903a-48c2-b96f-565267fb13b3",
 		"name": "Scribe API",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "30094260"
```

Comment:
What's `_exporter_id` here and do we actually need it?

Reply:
removed